hvpy/helpers.py

```diff
@@ -97,12 +97,15 @@ def createMovie(
     if res.get("error"):
         raise RuntimeError(res["error"])
     timeout_counter = time.time() + 60 * timeout  # Default 5 minutes
+    title = ""
     while True:
         status = getMovieStatus(
             id=res["id"],
             format=format,
             token=res["token"],
         )
+        if status.get("title"):
+            title = status["title"]
         if status["status"] in [0, 1]:
             time.sleep(3)
         if status["status"] == 2:
```

Comment:
I think `title = status.get("title")` should go here, after it completes.
You're checking for the title even if you know it's going to be `None` because status != 2.